src/bio/bio_context.c

```diff
@@ -440,6 +499,41 @@ bio_blob_create(uuid_t uuid, struct bio_xs_context *xs_ctxt, uint64_t blob_sz)
 	return rc;
 }
 
+int bio_mc_create(struct bio_xs_context *xs_ctxt, uuid_t pool_id, uint64_t meta_sz,
+		  uint64_t wal_sz, uint64_t data_sz, uint16_t csum_type, enum bio_mc_flags flags)
```

Comment:
'flags' isn't used? It'll be used in follow-on patch of supporting sysdb?

Reply:
Yes.